src/components/Reports/AdditionalSalaryRequest/Shared/formCompletionPercentageHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { CompleteFormValues } from '../CompleteForm/CompleteForm';
+
+export const formCompletionPercentageHelper = (
+  values: CompleteFormValues,
+): number => {
+  // Get all fields except defaultPercentage (checkbox doesn't count toward completion)
+  const fields = Object.entries(values).filter(
```

Comment:
Since we don't use the key anymore, this can be `Object.values(values)` and `([, value])` can become `(value)`.